mcd-sel-fj/scraped-data/[2022-08-10 23:51:04] mcd-sel-fj.csv

```diff
@@ -0,0 +1,41 @@
+,Date,Day,Territory,Menu Item,Price (FJD),Price (USD),Category,Menu
+1,2022/08/10,Wed,Fiji,Big Mac (S/R) -Beef,14.00,6.36,CLASSIC MCVALUE MEALS,Regular
+2,2022/08/10,Wed,Fiji,Cheeseburger (S/R) - Beef,10.20,4.63,CLASSIC MCVALUE MEALS,Regular
+3,2022/08/10,Wed,Fiji,Quarter Pounder (S/R) - Beef,14.45,6.57,CLASSIC MCVALUE MEALS,Regular
+4,2022/08/10,Wed,Fiji,Filet-O-Fish (S/R),11.70,5.32,CLASSIC MCVALUE MEALS,Regular
+5,2022/08/10,Wed,Fiji,McFeast (S/R) - Beef,15.80,7.18,CLASSIC MCVALUE MEALS,Regular
```

Comment:
Category names appear in uppercase because the webpage renders `h2` elements of each section as such.
This is not satisfactory.
Find alternative elements to scrape if possible.